.github/changelog-processor.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import os
+import sys
+import argparse
+
+parser = argparse.ArgumentParser(description="Process the CHANGELOG.md")
+parser.add_argument(
+    "changelog",
+    metavar="CHANGELOG",
+    help="Path to the CHANGELOG.md file"
```

Comment:
You can make this optional:

```suggestion
    help="Path to the CHANGELOG.md file",
    default="CHANGELOG.md",
    nargs='?'
```